datasets/common_voice/common_voice.py

```diff
@@ -0,0 +1,741 @@
+    'second_domain': "https://huggingface.co/great-new-dataset-second_domain.zip",
+}
+
+_FILES = {
+    "dev.tsv": "fields": {
+        "client_id": "client_id",	
+        "path": "path to sound file"	
+        "sentence": "sentence pronounced",
+        "up_votes": "upvotes by reviewers",
+        "down_votes": "down votes by reviewers"	
+        "age": "Age of the speaker":
+        "gender" "Gender of the speaker",
+        "accent": "accent of the speaker",
+        "locale": "location of the speaker",
+        "segment": "segment",
+    }
+    "invalidated.tsv": {
+        "client_id": "client_id",	
+        "path": "path to sound file"	
+        "sentence": "sentence pronounced",
+        "up_votes": "upvotes by reviewers",
+        "down_votes": "down votes by reviewers"	
+        "age": "Age of the speaker":
+        "gender" "Gender of the speaker",
+        "accent": "accent of the speaker",
+        "locale": "location of the speaker",
+        "segment": "segment"}
+    "other.tsv": 
+        {
+        "client_id": "client_id",	
+        "path": "path to sound file"	
+        "sentence": "sentence pronounced",
+        "up_votes": "upvotes by reviewers",
+        "down_votes": "down votes by reviewers"	
+        "age": "Age of the speaker":
+        "gender" "Gender of the speaker",
+        "accent": "accent of the speaker",
+        "locale": "location of the speaker",
+        "segment": "segment", }
+    "reported.tsv": {
+        {
+        "sentence": "sentence",	
+        "senetence_id": "sentence_id"	
+        "locale": "location of the speakr",
+        "reason": "reason for report"
+    }
+    "test.tsv": {
+         {
+        "client_id": "client_id",	
+        "path": "path to sound file"	
+        "sentence": "sentence pronounced",
+        "up_votes": "upvotes by reviewers",
+        "down_votes": "down votes by reviewers"	
+        "age": "Age of the speaker":
+        "gender" "Gender of the speaker",
+        "accent": "accent of the speaker",
+        "locale": "location of the speaker",
+        "segment": "segment", }
+    }
+    "train.tsv": {
+         {
+        "client_id": "client_id",	
```

Comment:
I think we should use those fields also as the corresponding `features = datasets.Features(` further down.
All `train.tsv`, `test.tsv` and `dev.tsv` has the same structure no?

Reply:
All files have the same structure, except reported, which is the one with the reason files are reported. However, I'm not sure it's as important to structure that data?